src/main.zig

```diff
@@ -577,6 +577,8 @@ const usage_build_generic =
     \\  --shared-memory                (WebAssembly) use shared linear memory
     \\  --global-base=[addr]           (WebAssembly) where to start to place global data
     \\  --export=[value]               (WebAssembly) Force a symbol to be exported
+    \\  -fentry                        (WebAssembly) Force output an entry point
+    \\  -fno-entry                     (WebAssembly) Do not output any entry point
```

Comment:
Let's combine this with `--entry`:

* `-fentry`: enable entry point with default name
* `-fentry=[name]`: override entry point to symbol name
* `-fno-entry`: no entry point

(remove `--entry`)

This can work for all targets, not only WebAssembly. link.Options already has:

```
    entry: ?[]const u8,
```

which you can use to represent all three cases. No need to add another field.

Reply:
Looks like this strategy only works for WebAssembly. I'll have to think of different logic to accomplish this.